### ISSUE TITLE
Consider dropping this disuasion

### ISSUE THREAD

**Issue by khionu** [cronokirby/alchemy#94]
Bots that do heavier tasks starting from a command may trigger typing to indicate they are processing the request. While I personally recommend using a "Please wait..." response that is then edited with the result, there is a sweet spot where the operation is too quick to send a interim message, but long enough to make people assume latency issues.

In short, it _can_ be used to add to the UX, and there's no reason to say bots shouldn't use it other than they don't actually type anything, which is a very narrow way of looking at the available features.

https://github.com/cronokirby/alchemy/blob/master/lib/client.ex#L700

**Comment by cronokirby** [comment]
IIRC there was a message similar to the one that exists now back then:
![image](https://user-images.githubusercontent.com/22821947/81954187-d83cf600-9608-11ea-9a46-6c35914dd9c8.png)
That was why I added that comment. Perhaps we should flesh out the comment more, or link back to discord's stance. It's kind of true that right now it sounds like there's no good reason to ever use it, but discord themselves weighs in a bit more on both sides

**Comment by khionu** [comment]
They do want to dissuade bots from using it more than on occasion, true, but they clearly don't want to fully prohibit it.

**Comment by cronokirby** [comment]
Agreed, we should make the language clearer, but I think mentioning what you and the discord docs mention is a good idea
